test/extended/image_ecosystem/mysql_replica.go

```diff
@@ -1,245 +0,0 @@
-package image_ecosystem
```

Comment:
a follow up ticket for these 2?

Reply:
Done: https://bugzilla.redhat.com/show_bug.cgi?id=1731892